### ISSUE TITLE
Descrever mais profundamente alguns pontos do código

### ISSUE THREAD

**Issue by joseliacosta** [training-center/sobre#69]
Após uma acalorada discussão no canal #general do Slack eu pensei que seria legal sugerir um `append` no código de conduta, principalmente no assunto tolerancia.

A Renata sugeriu esse link aqui e eu acho que é importante refletir sobre ele e tentar descrever no CoC que não toleramos intolerancia e porque não fazemos isso.
https://pt.m.wikipedia.org/wiki/Paradoxo_da_toler%C3%A2ncia

#paz ✌️

**Comment by khaosdoctor** [comment]
Concordo, mas acho que temos que pegar certinho a parte do artigo que tem que ser adicionada ao CoC, por exemplo, temos uma parte aqui:

> "Nós devemos portanto declarar, em nome da tolerância, o direito de não tolerar o intolerante."

De forma que o TC seria uma comunidade não tolerante por não tolerar o intolerante como dito na citação. O que poderia ser válido, mas ai não seriamos justos, conforme outra citação deste mesmo artigo:

> "[...] que uma sociedade justa deve tolerar o intolerante, do contrário, a sociedade seria então si própria intolerante, e portanto injusta [...]"

Então temos que ser mais tolerantes sob o risco de eliminar a tolerância como dito aqui:

> "Menos conhecido é o paradoxo da tolerância: tolerância ilimitada leva ao desaparecimento da tolerância."

Ou ser menos tolerantes, mas sob o risco de sermos injustos?

Acho que um meio termo seria uma coisa assim:

> (...)Michael Walzer indagou: "Devemos nós tolerar os intolerantes?". (...) a maioria dos grupos religiosos minoritários que são beneficiários da tolerância, são eles próprios intolerantes, ao menos em alguns aspectos. Em um regime tolerante, estas pessoas podem ter que aprender a tolerar, ou ao menos comportar-se "como se possuíssem esta virtude"

O que acham?

**Comment by jaqueventurim** [comment]
pra mim o que deve ser posto no CoC é isso aqui: 

> "...acreditam que sua própria segurança, e daquelas instituições da liberdade, estão em perigo".

lembrando sempre que segurança NÃO É só a física. nossa saúde mental entra nisso, e quando uma mulher/um LGBT/uma pessoa negra/uma pessoa trans diz que já está exausta de discutir tal coisa é porque a segurança dela já foi destruída, e portanto, está em perigo.

**Comment by jaqueventurim** [comment]
outra coisa extremamente importante também é que o CoC **não pode ficar subentendido** então nada de ficar em cima do muro. ele tem que ser objetivo e simples.

**Comment by woliveiras** [comment]
Gente, sem filosofar muito, é isso aqui que importa pra uma comunidade (não só para o TC):

> se não estivermos preparados para defender a sociedade tolerante **contra a investida dos intolerantes**, então os tolerantes serão destruídos, e a tolerância junto destes

Investida dos intolerantes.

Nós precisamos estar preparados para defendermos as pessoas da nossa comunidade contra uma pessoa intolerante. Não vejo o que filosofar ou discutir quanto a isso.

Todo o resto é filosofia que pode ser subentendida. Como "ser intolerante contra a intolerância"... Isso pode ser levado por vários contextos filosóficos até chegar ao ponto de que todo mundo é intolerante e pronto. 😂 

Só que isso já está no COC: 

> **Não toleramos** nenhum tipo de repúdio, humilhação ou discriminação

E depois aprofunda colocando exatamente o que é ruim em uma comunidade, justamente para não ser subjetivo/abstrato demais:

>  Gênero, identidade de gênero ou expressão de gênero;
> Orientação sexual;
> Restrições físicas;
> Aparência física (incluindo, mas não limitado, ao tamanho do corpo);
> Raça e/ou etnia;
> Idade;
> Religião ou a falta dela;
> Escolha de tecnologias;

Podemos colocar ali, também, posição política, visto a última treta que deu na comunidade.

Uma pessoa de esquerda não deve arrumar confusão com alguém de direita e vice-versa (como se só existissem esses dois posicionamentos, mas, infelizmente, as pessoas estão politizadas via Facebook, portanto leiam isso só como exemplo, viu).

Mais pra frente o COC reforça:

> Nós adotamos a política de **tolerância zero** para assédios, perseguições ou discriminações;

E podemos colocar ali também que nós devemos evitar discussão política, visto que inclusão independe de posicionamento político e poderemos continuar falando disso lindamente, mas não será permitido discussão de direita vs esquerda na comunidade.

Concordam comigo que já existe proteção contra intolerantes no grupo e com colocar as clausulas sobre política?

Só precisamos que a comunidade adote a prática de denunciar, pois o form de denuncia existe e não está sendo usado quando acontece algo. 😞

**Comment by joseliacosta** [comment]
Então @woliveiras é que eu to percebendo que a nuance de agressividade que acontece nessas discussões é muito sutil... E perceba que quando alguém coloca um comentário intolerante e parece irredutivel vai mó galera discutir com a pessoa, mó galera fica exausta de sempre ter que explicar as mesmas coisas (que sim, parecem bem explicitas no CoC e nos textos do Medium do TC) e no final sempre sai alguém nervoso porque tentamos ser "tolerantes". 
O que eu achei importante desse paradoxo é entender que não podemos dar lugar para alguém que alimente uma discussão intolerante e esse é meu ponto pra deixar isso explicito e com detalhes (perdoe-me a redundância) no código de conduta, entende? 
Eu sei que somos uma comunidade que passa segurança para muitas pessoas e que respeita o código de conduta, porém, como vc mesmo disse o form de denuncia existe e não é usado, mas o form de denuncia é meio que pra depois que alguma coisa aconteceu :(
Penso que ainda que utópico acreditar que todas as pessoas leram e entenderam de primeira o CoC isso pode nos deixar mais protegidos para discursos intolerantes porém com outra roupagem.
O que vc acha?

**Comment by alinebastos** [comment]
> E perceba que quando alguém coloca um comentário intolerante e parece irredutivel vai mó galera discutir com a pessoa, mó galera fica exausta de sempre ter que explicar as mesmas coisas (que sim, parecem bem explicitas no CoC e nos textos do Medium do TC) e no final sempre sai alguém nervoso porque tentamos ser "tolerantes".

Concordo, é cansativo demais!!! Precisamos de uma regra clara pra evitar essas discussões que geralmente não dão em nada, só stress.

**Comment by lbighetti** [comment]
Uma solução alternativa que funciona pra gente no Telegram do Elixir é deletar a mensagem em questão pra não dar espaço pra gerar mais intolerância, e nem gerar mais perturbação além das pessoas iniciais que leram. Daí mandar msg pra pessoa no privado do tipo: "Cara, apaguei sua msg por isso".

Isso anda funcionando bem e podíamos tentar aqui.

Por exemplo, duas pessoa se estranharam lá e uma delas fez um comentário que dava pra ser interpretado como homofóbico. Não era explicitamente, e a pessoa em questão poderia tentar dialogar e se defender - parecido com esse caso. Deletei e mandei msg pros dois individualmente explicando que gostaríamos de ser o mais inclusivo possível e evitar esse tipo de comentário. 

Benefícios:  
- O resto do grupo não se afetou, apenas as pessoas em questão e os admins (talvez algumas poucas que leram antes de eu deletar?)
- Não fica lá na cara de todo mundo thread / tópico, o que é propício a perdurar a discussão desgastante e gerar mais comentários similares ao problemático.
- As pessoas no geral escutam e refletem quando falamos com elas em privado, ao contrário de apontar o erro delas em público. No caso do TC o cara era um babaca mesmo, mas muita gente tá disposta a mudar se a gente conversar, e a gente perde isso isolando a pessoa em público.


Contras:  
- As pessoas podem se sentir "censuradas". 

O processo pode ser:  
- Alguém moderação vê o comentário, manda no channel do slack e deleta.
- Manda msg pra(s) pessoa(s) em privado.

O que acham ?

**Comment by woliveiras** [comment]
Concordo 100%, @joselialcosta!!!

Mas nem precisa alterar o COC pra isso, só precisamos mesmo fazer com que se cumpra e colocar um limite de até onde tentamos dialogar com alguém...

O objetivo do TC não é, de fato, orientar pessoas intolerantes, mas auxiliar pessoas a entrarem na área de desenvolvimento de software e por isso é fora de escopo ficar discutindo, realmente.

Essas discussões só acontecem por culpa minha viu. Me desculpem. É porque eu queria que ajudássemos os movimentos a mudarem a cabeça das pessoas e fiquei forçando a galera do core a discutir mais antes de expulsar alguém, mas na comunidade (online) não é o melhor lugar para se fazer isso, como já constatamos. :joy: 

O que precisamos é definir um limite aqui: até que ponto seguimos em uma discussão com alguém que demonstra ser intolerante ou sem conhecimento sobre as injustiças? Porque a falta de conhecimento pode levar alguém a cometer erros contra o código de conduta e eu acredito que nós deveríamos auxiliar essa pessoa a chegar a esse conhecimento.

**Comment by woliveiras** [comment]
A alternativa do @lbighetti eu acho uma excelente ideia, gente!

**Comment by alinebastos** [comment]
> Essas discussões só acontecem por culpa minha viu. Me desculpem. É porque eu queria que ajudássemos os movimentos a mudarem a cabeça das pessoas e fiquei forçando a galera do core a discutir mais antes de expulsar alguém, mas na comunidade (online) não é o melhor lugar para se fazer isso, como já constatamos.

É verdade, é tua culpa! hahahaha
Que isso não se repita :P

**Comment by khaosdoctor** [comment]
> A alternativa do @lbighetti eu acho uma excelente ideia, gente!

Também gosto dessa alternativa, dessa forma não oneramos a comunidade como um todo e sim só aqueles poucos que lerem.

Nos contras eu não acho que seja uma censura, porque se está gerando intrigas e ameaçando dividir as pessoas do grupo, então é meio que uma "autopreservação", para manter a paz geral.
